lib/src/tasks/test/api.dart

```diff
@@ -18,17 +18,30 @@ import 'dart:async';
 
 import 'package:dart_dev/util.dart' show TaskProcess;
 
+import 'package:dart_dev/src/tasks/serve/api.dart' as serve;
 import 'package:dart_dev/src/tasks/task.dart';
 
-TestTask test(
-    {int concurrency,
+Future<TestTask> test({int concurrency,
+    bool pubServe,
     List<String> platforms: const [],
-    List<String> tests: const []}) {
+    List<String> tests: const []}) async {
+  serve.PubServeTask pubServeTask;
+  int pubPort = null;
+  if (pubServe) {
+    // Start `pub serve` on the `test` directory
+    pubServeTask = serve.pubServe(additionalArgs: ['test']);
+    var serveInfo = await pubServeTask.serveInfos.first;
```

Comment:
I'd like to log some output here... At the very least, an indication that the pub server is starting, but ideally the `pub serve` output up until it starts.

Should `pubServeTask` be a field on `TestTask`, so that it can be accessed and logged accordingly in `test/cli.dart`?